hack/docker-builder/Dockerfile

```diff
@@ -18,7 +18,10 @@ RUN \
     source /etc/profile.d/gimme.sh && \
     go get github.com/mattn/goveralls && \
     go get -u github.com/Masterminds/glide && \
-    go get -u golang.org/x/tools/cmd/goimports && \
+    go get golang.org/x/tools/cmd/goimports && \
```

Comment:
I think we also have add `-d`, to be most protected against breakage. Otherwise it would try to  compile master too.

Reply:
Yeah I even intended to, checked the help message... and forgot. :) Fixed now.